packages/ag-charts-community/src/chart/polarChart.ts

```diff
@@ -80,7 +79,7 @@ export class PolarChart extends Chart {
     }
 
     private async computeCircle(seriesBox: BBox) {
-        const polarSeries = this.series.filter((series): series is PolarSeries<SeriesNodeDatum> => {
+        const polarSeries = this.series.filter((series): series is PolarSeries<any, any> => {
```

Comment:
any?

Reply:
Unfortunately needed as there's no way to derive it, and `never` and `unknown` cause issues later in this function :-(